media/css/dokieli.css

```
@@ -28,14 +28,23 @@ margin-top:0 !important;
 }
 }
 
-@media screen and (min-width: 75em){
-aside.note article {
-position: absolute !important;
-left: auto;
-right: -27%;
-z-index: 1;
-width: 25%;
+@media screen and (max-width: 700px){
+  #document-menu {
+    z-index: 1 !important;
+  }
+  #document-items {
+    z-index: 0 !important;
+  }
```

```suggestion
  #document-items {
    z-index: 2147483646 !important;
  }
```

As far as I know, the highest z-index value is 2147483647 and that's set on `.do.on` in which `#document-menu` and `#document-items` inherit. Setting it to 1 and 0 here lowers the z-index than what's intended. The suggested change may be closer to what you've intended.

---

Also note that with this change also blocks the user from reaching document-items altogether. So, perhaps we need a different approach. At the moment document-items is indeed on the way of the menu but at least it can be closed and the menu can be reached after whereas with the change here, the menu closes and items goes with it.

---

This is what I meant by "other uses of z-index". Why are we setting to a number such as 2147483647 in the first place?

---

> At the moment document-items is indeed on the way of the menu but at least it can be closed and the menu can be reached after whereas with the change here, the menu closes and items goes with it.

That's right. Would a better approach be moving it further down in the screen?

---

Claiming the top space in the view that's closest to the user - perhaps the layer between the user agent and the document. It goes over the stylesheets specific to the document as well as other browser extensions.

---

Then maybe we could have it collapsed only on mobile, and display a button to expand (like TOC in https://solid.github.io/webid-profile/, for example).

---

For now, I think I will close this and make a separate PR with different e2e tests.